### PR TITLE
Add Adam

### PR DESCRIPTION
**Pull request by Adam0Brien**

### CODE REVIEW

**Comment by travier** [comment]
/ok-to-test
@Adam0Brien Can you rebase this one?

**Review by c4rt0 — APPROVED** [review]
/lgtm

File: OWNERS [hunk 1]
```diff
@@ -31,3 +31,4 @@ approvers:
   - jschintag
   - c4rt0
   - cverna
+  - Adam0Brien
```
Comment (travier) [inline OWNERS:34]: This is now missing an empty line at the end of the file.

**Review by travier — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (Adam0Brien) [reply OWNERS]: added the empty line

**Comment by travier** [comment]
You now have a line with a few space characters and thus not empty.

You should configure your editor to always make sure there is an empty line at the end of files.

**Comment by Adam0Brien** [comment]
> You now have a line with a few space characters and thus not empty.
> 
> You should configure your editor to always make sure there is an empty line at the end of files.

set up vscode for the newlines, added and pushed again

**Comment by travier** [comment]
OK, LGTM now
/lgtm

**Pull request merged**